bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/AmcrestHandler.java

```diff
@@ -0,0 +1,248 @@
+                }
+                return;
+            case CHANNEL_API_ACCESS:
+                if (command.toString() != null) {
+                    ipCameraHandler.logger.info("API Access was sent this command :{}", command.toString());
+                    ipCameraHandler.sendHttpGET(command.toString());
+                    ipCameraHandler.setChannelState(CHANNEL_API_ACCESS, StringType.valueOf(""));
+                }
+                return;
+            case CHANNEL_ENABLE_LED:
+                ipCameraHandler.setChannelState(CHANNEL_AUTO_LED, OnOffType.valueOf("OFF"));
+                if ("0".equals(command.toString()) || "OFF".equals(command.toString())) {
+                    ipCameraHandler.sendHttpGET("/cgi-bin/configManager.cgi?action=setConfig&Lighting[0][0].Mode=Off");
+                } else if ("ON".equals(command.toString())) {
+                    ipCameraHandler
+                            .sendHttpGET("/cgi-bin/configManager.cgi?action=setConfig&Lighting[0][0].Mode=Manual");
+                } else {
+                    ipCameraHandler.sendHttpGET(
+                            "/cgi-bin/configManager.cgi?action=setConfig&Lighting[0][0].Mode=Manual&Lighting[0][0].MiddleLight[0].Light="
+                                    + command.toString());
+                }
+                return;
+            case CHANNEL_AUTO_LED:
+                if ("ON".equals(command.toString())) {
+                    ipCameraHandler.setChannelState(CHANNEL_ENABLE_LED, UnDefType.valueOf("UNDEF"));
+                    ipCameraHandler.sendHttpGET("/cgi-bin/configManager.cgi?action=setConfig&Lighting[0][0].Mode=Auto");
+                }
+                return;
+            case CHANNEL_THRESHOLD_AUDIO_ALARM:
+                int threshold = Math.round(Float.valueOf(command.toString()));
+
+                if (threshold == 0) {
+                    ipCameraHandler.sendHttpGET(
+                            "/cgi-bin/configManager.cgi?action=setConfig&AudioDetect[0].MutationThreold=1");
+                } else {
+                    ipCameraHandler.sendHttpGET(
+                            "/cgi-bin/configManager.cgi?action=setConfig&AudioDetect[0].MutationThreold=" + threshold);
+                }
+                return;
+            case CHANNEL_ENABLE_AUDIO_ALARM:
+                if ("ON".equals(command.toString())) {
+                    ipCameraHandler.sendHttpGET(
+                            "/cgi-bin/configManager.cgi?action=setConfig&AudioDetect[0].MutationDetect=true&AudioDetect[0].EventHandler.Dejitter=1");
+                } else {
+                    ipCameraHandler.sendHttpGET(
+                            "/cgi-bin/configManager.cgi?action=setConfig&AudioDetect[0].MutationDetect=false");
+                }
+                return;
+            case CHANNEL_ENABLE_LINE_CROSSING_ALARM:
+                if ("ON".equals(command.toString())) {
+
+                } else {
+
+                }
```

Comment:
missing code?

Reply:
Thank you, copy paste bandit struck. Fixed.